.github/workflows/nightly.yml

```diff
@@ -291,7 +304,33 @@ jobs:
           docker build -t test-image app/tomee/jakartaee-hello-world
           docker rmi test-image
           rm -rf app/tomee
+          
+      - name: Run Archetype for EE 9, SE 8, TomEE
```

Comment:
This test should be using the Web profile. That is the only profile that TomEE supports. For Jakarta EE 9, it should also be using Java SE 11. TomEE 9 does not support Java SE 8.